src/coreclr/src/jit/importer.cpp

```diff
@@ -2117,11 +2117,17 @@ GenTree* Compiler::impRuntimeLookupToTree(CORINFO_RESOLVED_TOKEN* pResolvedToken
                                       nullptr DEBUGARG("impRuntimeLookup indirectOffset"));
         }
 
+        // The last indirection could be subject to a size check (dynamic dictionary expansion)
+        bool isLastIndirectionWithSizeCheck = (i == pRuntimeLookup->indirections - 1 && pRuntimeLookup->sizeOffset != CORINFO_NO_SIZE_CHECK);
```

Comment:
We usually fully parenthesize:
```suggestion
        bool isLastIndirectionWithSizeCheck = ((i == pRuntimeLookup->indirections - 1) && (pRuntimeLookup->sizeOffset != CORINFO_NO_SIZE_CHECK));
```
Also, will likely require clang-formatting.

Reply:
I have fixed the formatting. I'll look into clang formatting and providing the assembly diffs as the next step.